src/app/analyze/analyze-overview/analyze-overview.component.html

```diff
@@ -146,6 +146,215 @@ <h2 class="panel-title text-center">Hypotheses Evaluating</h2>
+            <p class="margin-bottom-0">Description of the "SPACE" goes here.</p>
+            <p class="margin-bottom-0">Space owner would write the description here...</p>
+            <p>It also has edit option for him.</p>
+          </div>
+        </div>
+      </aside>
+    </div>
+    <div class="flex-grow-8">
+      <aside>
+        <div class="card-pf">
+          <div class="flex-grow-2">
+            <h2 class="card-pf-title">Analytical Report</h2>
+            <form class="padding-top-15">
+              <div class="form-group">
+                <div class="combobox-container">
+                  <input type="hidden" name="" value="">
+                  <div class="input-group">
+                    <input type="text" autocomplete="false" placeholder="Select your build/pipeline" class="combobox form-control">
+                    <span class="input-group-addon dropdown-toggle" data-dropdown="dropdown">
+                      <span class="caret"></span> <span class="glyphicon glyphicon-remove"></span>
+                    </span>
+                  </div>
+                </div>
+                <select class="combobox form-control hide">
+                  <option value="" selected="selected">Select your build/pipeline</option>
+                  <option value="AL">Pipeline 1</option>
+                  <option value="AK">Pipeline 2</option>
+                  <option value="AZ">Pipeline 3</option>
+                  <option value="AR">Pipeline 4</option>
+                  <option value="CA">Pipeline 5</option>
+                  <option value="CO">Pipeline 6</option>
+                  <option value="CT">Pipeline 7</option>
+                  <option value="DE">Pipeline 8</option>
+                </select>
+              </div>
+            </form>
+            <ul class=" analytical-report-list">
+              <li>
+                <p>
+                  <a>Build #102</a>
+                  <span> - You have used</span>
+                  <a>Java JDK 2.0.3</a>
+                  <span>better way to run/deploy you can use</span>
+                  <a>Java JDK 2.0.8</a>
+                </p>
+              </li>
+              <li>
+                <p>
+                  <a>Build #102</a>
+                  <span> - We recommend to use you can use</span>
+                  <a>Java JDK 2.0.8</a>
+                </p>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </aside>
+    </div>
+  </section>
+  <section class="row row-cards-pf space-overview-wrapper">
+    <div class="flex-grow-4 " >
+      <aside>
+        <div class="card-pf">
+          <div class="flex-grow-1 flex-container in-column-direction">
+            <h2 class="card-pf-title card-pf-utilization-details">
+              <span>My Work items</span>
+              <span class="badge">20</span>
+            </h2>
+            <div class="analyze-overview-add-items">
+              <div class="card-pf-body text-center">
```

Comment:
The blank state of a card should use this layout/styling: http://www.patternfly.org/pattern-library/communication/empty-state/#/code.